README.md

```diff
@@ -91,16 +91,27 @@ plugins:
 ```
 
 by default it will search in the folder where your mkdocs.yml is kept
-and in the docs folder for another folder called `_data`
+and in the docs folder for another folder called `_data` 
+(i.e. `./docs/_data/site.yaml`), available as `{{ site.whatever_variable_in_the_yaml}}`.
+
+If these paths are found, the plugin will read all `.yml|.yaml` and `.json`
+files inside them and add the data in them under the `extra` key.
+
+For example, if you have a file called `[path/to/datafiles/]sections/captions.yaml` 
+which includes a variable `foo` - where `[path/to/datafiles/]` is the path declared
+in your configuration under `data` - the data inside that file will be available in
+ your templates as `{{sections.captions.foo}}` or `{{sections['captions']['foo']}}`.
+
+Alternatively, you can access all files and variable declared under `data` in template
+using `extra` key. 
+This is particularly useful if your folder or filename do not comply with the Python
+variable naming rules.
+For example, if you have a file `[path/to/datafiles/]1_example/captions.yaml`
```

Comment:
maybe just 

```
path/to/datafiles/1_example/captions.yaml
```

the [] makes difficult to parse

Reply:
I wrote it with `[]` to keep it consistent with the way you had already written the documentation. Are you suggesting removing it on line 109 but keeping it on lines 100 and 101?

Reply:
right you are, strange i dont usually use that notation. naja!